### PR TITLE
Wagtail 5.1

### PR DESCRIPTION
**Pull request by katdom13**
**Upstream PR:** https://github.com/neon-jungle/wagtail-accessibility/pull/13

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[tox]
+minversion = 3.14.6
+envlist =
+    py{38,39,310,311}-dj{32,41}-wagtail{41,50,51}
+    py{39,310,311}-dj{42}-wagtail{50,51}
```
Comment (nickmoreton) [inline tox.ini]: I think here we should take into account that dj32 doesn't support py3.11

File: setup.py [hunk 2]
```diff
@@ -13,6 +13,18 @@
     use_setuptools()
     from setuptools import setup, find_packages
 
+install_requires = [
+    'wagtail>=2.9',
```
Comment (nickmoreton) [inline setup.py]: As the maintainer probably won't be considering this work we should use.

```suggestion
    'wagtail>=4.1,
```

**Review by nickmoreton — CHANGES_REQUESTED** [review]
Hi @katdom13 I left a couple of changes to consider. Thanks

File: tox.ini (hunk 1 above)
Comment (katdom13) [reply tox.ini]: Sorry about that, I patterned it here: https://github.com/torchbox/wagtail-markdown/pull/130/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R5-R6

Should be fixed here: https://github.com/torchbox-forks/wagtail-accessibility/pull/1/commits/cfd12bc91952ccf4856d210c5d91d3dd92c3e654

File: setup.py (hunk 2 above)
Comment (katdom13) [reply setup.py]: Fixed here: https://github.com/torchbox-forks/wagtail-accessibility/pull/1/commits/d153b1d99478b27b1d305935c4410d8a92437842

File: tox.ini (hunk 1 above)
Comment (nickmoreton) [reply tox.ini]: Ah OK, thanks for updating it. I wonder what's right now but I don't suppose it matters too much 😄

File: setup.py (hunk 2 above)
Comment (nickmoreton) [reply setup.py]: Excellent 👍

**Review by nickmoreton — APPROVED** [review]
Thanks @katdom13 I approved it.

**Pull request merged**